models/repo.go

```diff
@@ -392,6 +392,19 @@ func (repo *Repository) UnitEnabled(tp UnitType) bool {
 	return false
 }
 
+// AnyUnitEnabled if this repository has the any of the given units enabled
+func (repo *Repository) AnyUnitEnabled(tps ...UnitType) bool {
+	repo.getUnits(x)
```

Comment:
There is also `getUnitsByRepoIDAndIDs` (currently unused anywhere)

Reply:
As we already discussed in discord it is better to keep same behaviour as in `UnitEnabled`